lib/file-perms.h

```diff
@@ -51,5 +51,6 @@ gboolean file_perm_options_apply_file(const FilePermOptions *self, gchar *name);
 gboolean file_perm_options_apply_dir(const FilePermOptions *self, gchar *name);
 gboolean file_perm_options_apply_fd(const FilePermOptions *self, gint fd);
 gboolean file_perm_options_create_containing_directory(const FilePermOptions *self, gchar *name);
+gboolean file_perm_options_check_file_is_device(gchar *name);
```

Comment:
Please, use `const gchar *` as type for `name`. (name is used only for passing it to `stat` and `stat` expects `const char *`).